test/test_plugin_pushbullet.py

```diff
@@ -181,6 +181,18 @@
 )
 
 
+@pytest.fixture(scope="module", autouse=True)
+def no_throttling(module_mocker):
+    """
+    A pytest fixture which disables Apprise throttling for `NotifyPushBullet`.
+
+    This fixture, in contrast to the `no_throttling` fixture in `conftest.py`,
+    is both module-scoped and automatically applied. In this manner, it is not
+    needed to attach it to individual test cases within this module.
+    """
+    module_mocker.patch.object(NotifyPushBullet, "request_rate_per_sec", 0)
```

Comment:
This implementation was the result of our conversation at https://github.com/caronc/apprise/pull/687#discussion_r992312635, thanks a stack.

> This fixture is both module-scoped and automatically applied. In this manner, it is not needed to attach it to individual test cases within this module.

Saying that, it is still possible to adjust that behavior for specific test cases. For example, if you would have a test module where one specific test case needs to run with throttling enabled, you can just slap `mocker` on your test case function, and patch the target into another direction, like
```
mocker.patch.object(NotifyPushBullet, "request_rate_per_sec", 42.42)
```

Reply:
The details as to what is best, I'm at the point to just let you decide. My only concern was that it be nestled in with the test file associated with the object being tested and you nailed that :rocket: Your approach (while more code) is less harsh then my monkey patching where i just fixed the request_rate_per_sec in each test.  This monkey patching still takes place in `test/helpers/rest.py` [here](https://github.com/caronc/apprise/blob/cb7f51d82a55910a1b6b4fb6adcfe9ec2835a407/test/helpers/rest.py#L476), [here](https://github.com/caronc/apprise/blob/cb7f51d82a55910a1b6b4fb6adcfe9ec2835a407/test/helpers/rest.py#L391), [here](https://github.com/caronc/apprise/blob/cb7f51d82a55910a1b6b4fb6adcfe9ec2835a407/test/helpers/rest.py#L234), and [here](https://github.com/caronc/apprise/blob/cb7f51d82a55910a1b6b4fb6adcfe9ec2835a407/test/helpers/rest.py#L119).  

Personally it doesn't bother me at all, but i can just imagine you twitching with me sharing that with you after all of the re-writing and precautionary (and proper) code you've been installing over the past few days :wink:

Reply:
As a followup to what we have here, I will add your suggestion from https://github.com/caronc/apprise/pull/687#issuecomment-1279338634:

>> Honestly though, if your `disable_throttle` just iterated through `common.NOTIFY_CUSTOM_MODULE_MAP` and set the values to zero for the decorator. Thoughts?
>
> I see that you are in favor of such a catch-all fixture. I will be happy to add one on behalf of #689, because I would like to use `pytest-mock` aka. `mocker` for that job. Would you accept this as a promise for the future?

Reply:
Did you have something like ea4d7e11b in mind? In this manner, disabling throttling works out of the box, and it seems to work well so far.

Reply:
Hi again. I've diverted this into a separate patch, see #701.